src/sentry/api/endpoints/organization_pinned_searches.py

```diff
@@ -13,6 +14,9 @@
 from sentry.models.search_common import SearchType
 
 
+PINNED_SEARCH_NAME = 'My Pinned Search'
```

Comment:
Should this be translated?

Reply:
No idea, do we translate the global search names?

Reply:
Probably not because we jam them in the database.